test/workload.yaml

```diff
@@ -10,8 +10,12 @@
   spec:
     interface: eth0
     profiles: [prof1, prof2]
-    ipNetworks: [1.2.3.4/32, "00:bb::aa/10"]
+    ipNetworks: ["1.2.3.4/32", "00:bb::aa/128"]
+    ipNATs:
+    - externalIP: 10.20.30.40
```

Comment:
the resource is a workload endpoint so internal and external would be relative to the endpoint...

i'll do a little poll.